src/Workflow.py

```diff
@@ -0,0 +1,72 @@
+import streamlit as st
+from .workflow.WorkflowManager import WorkflowManager
+from .workflow.Files import Files
+
+class Workflow(WorkflowManager):
```

Comment:
Can you add more documentation to this class? I think I would still have trouble to set up something that is not very close to this example.

e.g. I think it should be clearer which methods are for creating the frontend part (e.g., setting up the pages)
would it make sense to rename upload to createUploadPage ? As I understand, it fills the content with the streamlit components ...

Reply:
I will add some more in a docstring here. Does the `pages/6_📖_TOPP-Workflow_Docs.py` page help?